kwasm-lemmas.md

```diff
@@ -13,24 +13,28 @@ Basic arithmetic
 ----------------
 
 ```k
-    rule (0 +Int X) => X
-    rule (X +Int 0) => X
+    rule (0 +Int X) => X [simplification]
+    rule (X +Int 0) => X [simplification]
 
     rule (X +Int (Y *Int N)) modInt N => X modInt N
       requires N >=Int 1
+      [simplification]
     rule (X +Int (Y *Int N)) /Int N   => (X /Int N) +Int Y
       requires N =/=Int 0
+      [simplification]
+
+    rule X modInt N => X
+      requires X <=Int 0
```

Comment:
Should this be `X >=Int 0`

Reply:
Good catch. I'll add a regression tests to `simple-arithmetic-spec`.